CHANGES.md

```diff
@@ -1,21 +1,8 @@
 Change log
 ==========
 
-1.1.0-rc2 (2015-01-29)
```

Comment:
We weren't planning on it, but the git tags will stick around, so that'll help when narrowing things down.